src/robusta/core/sinks/transformer.py

```diff
@@ -50,7 +50,7 @@ def __markdown_to_html(cls, mrkdwn_text: str) -> str:
         # replace slack markdown bold: from *bold text* to <b>bold text<b>  (markdown2 converts this to italic)
         mrkdwn_text = re.sub(r"\*([^\*]*)\*", r"<b>\1</b>", mrkdwn_text)
 
-        # Note - markdown2 should be used after slack links already converted, other wise it's getting corrupted!
+        # Note - markdown2 should be used after slack links already converted, otherwise it's getting corrupted!
```

Comment:
This wasn't flagged by the spell checker (although I think I'll add it to forbidden since I can't think of a case when it'd be correct... the feature is only a couple of weeks old) but I noticed it while trying to review each instance of `it's` and `its`.

I think I'd probably suggest changing `it's getting` to `it will be`, but that's far beyond the scope of a spelling PR.